templates/coreos.yaml

```diff
@@ -0,0 +1,110 @@
+---
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
Might be good to have an example that can be used out of the box? so without the discovery?